pkg/sql/bench_test.go

```diff
@@ -332,6 +332,39 @@ func runBenchmarkInsertFK(b *testing.B, db *gosql.DB, count int) {
 	}
 }
 
+// runBenchmarkInsertSecondaryIndex benchmarks inserting count rows into a table with a
+// secondary index.
+func runBenchmarkInsertSecondaryIndex(b *testing.B, db *gosql.DB, count int) {
+	defer func() {
+		if _, err := db.Exec(`DROP TABLE IF EXISTS bench.insert`); err != nil {
+			b.Fatal(err)
+		}
+	}()
+
+	if _, err := db.Exec(`CREATE TABLE bench.insert (k INT PRIMARY KEY, v INT , INDEX(v))`); err != nil {
```

Comment:
I think you should make several indexes on this column, so that the benchmark is affected as much as possible by the index entry creation code. You can do that with a `CREATE INDEX` statement, which can be run on the same column as many times as you want.